asm.S

```diff
@@ -94,6 +94,10 @@ Reset:
 #endif
 
 4:
+.cfi_def_cfa sp, 0
```

Comment:
This might be worth an explanation, as it's not something you often come across in ARM asm.

Maybe link to https://sourceware.org/binutils/docs/as/CFI-directives.html, or include their note:

> `.cfi_def_cfa` defines a rule for computing CFA as: take address from register and add offset to it.

Reply:
Added a comment and link